app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -306,6 +397,69 @@
       </div>
     </div>
   {{/if}}
+
+  {{#if this.isAddingMainSigns}}
+    <div class="au-o-grid__item au-u-2-5@medium">
+      <div
+        class="au-c-body-container au-c-action-sidebar"
+        {{auto-focus ".au-c-action-sidebar__header"}}
+      >
+        <div class="au-o-box au-c-action-sidebar__header" tabindex="0">
+          <AuToolbar class="au-u-margin-bottom-small">
+            <AuToolbarGroup>
+              <AuHeading @level="2" @skin="3">
+                {{t "mainSign.crud.add"}}
+              </AuHeading>
+            </AuToolbarGroup>
+            <button
+              type="button"
+              class="au-c-close au-c-close--large"
+              {{on "click" this.toggleAddMainSigns}}
+            >
+              <AuIcon @icon="cross" @size="large" />
+              <span class="au-u-hidden-visually">
+                {{t "utility.close"}}
+              </span>
+            </button>
+          </AuToolbar>
+          <AuContent @skin="small">
+            <AuLabel>
+              {{t "mainSign.attr.category"}}
+            </AuLabel>
+            <PowerSelect
+              @allowClear={{true}}
+              @searchEnabled={{true}}
+              @noMatchesMessage={{t "roadSignConcept.crud.noData"}}
+              @searchField="label"
+              @options={{@model.categories}}
+              @selected={{this.category}}
+              @onChange={{this.handleCategorySelection}} as |category|
```

Comment:
Should this use the same dataset as the related sidebar? This action does some filtering which might need to be adjusted for this?

Reply:
The thinking behind this is that a "main" and "related" are mutually exclusive. I've adapted the filter now to reflect this.